transform_encoder.py

```diff
@@ -119,11 +119,9 @@ def _generate_input_tokens(self, texts):
         return input_tokens
 
     def _get_docs_batch_generator(self, docs: DocumentArray, parameters: Dict):
-        traversal_paths = parameters.get('traversal_path', self.default_traversal_path)
+        traversal_path = parameters.get('traversal_path', self.default_traversal_path)
```

Comment:
traversal_paths should be plural both the variable and the parameter

Reply:
Check this PR for usage: https://github.com/jina-ai/executor-indexers/pull/25